qiskit_experiments/library/characterization/analysis/ramsey_xy_analysis.py

```diff
@@ -37,28 +37,28 @@ class RamseyXYAnalysis(curve.CurveAnalysis):
         defpar \rm amp:
             desc: Amplitude of both series.
             init_guess: Half of the maximum y value less the minimum y value.
-            bounds: [-2, 2] scaled to the maximum signal value.
+            bounds: [0, maximum absolute Y data]
```

Comment:
To match the code this should be:

```suggestion
            bounds: [0, inf]
```

Reply:
Updated accordingly